lang/main-sv.json

```diff
@@ -1211,11 +1276,13 @@
         "talkWhileMutedPopup": "Försöker du tala? Din mikrofon är tystad.",
         "tileViewToggle": "Öppna eller stäng panelvyn",
         "toggleCamera": "Byta kamera",
+        "unmute": "Slå på ljud",
         "videoSettings": "Video inställningar",
-        "videomute": "Aktivera / avaktivera kameran"
+        "videomute": "Aktivera / avaktivera kameran",
+        "videounmute": "Stoppa kameran"
     },
     "transcribing": {
-        "ccButtonTooltip": "Starta / Avsluta undertexter",
+        "ccButtonTooltip": "Atkivera / avaktivera undertexter",
```

Comment:
Hello, I just stumbled on this PR and noticed that you have a typo here 😊

Reply:
Well spotted! Thanks!